docs/contributors/opentracingprovider.adoc

```diff
@@ -188,4 +189,34 @@ When the ‘inject’ method is called on the OpenTracing Tracer, it will provid
 An unique id should be created, associated with the trace state being returned for injection into the outbound message, and used to create an `Interaction` based correlation id for the current node. The node should also be created as a `Producer`.
 
 
+=== Sampling 
 
+Sampling implementation is not required, however if a provider decideds to implement it should follow following rules.
```

Comment:
We need to distinguish between whether an opentracing provider should support sampling vs whether the application needs to provide a sampling configuration. I think we should ensure all the APM OT providers support sampling using the same common API - however the application does not need to provide it, and if they don't provide, (and a reporting level is not propagated) then it should default to All.